client/main.html

```diff
@@ -17,18 +17,11 @@
   {{#if currentUser}}
   <div class="formular">{{> addPlayerForm}}</div>
   <ul>
-    {{#each player}}
-    <li class="player {{selectedClass}}">{{name}}: {{score}}</li>
-
+    {{#each player in players}}
+    <li class="player {{selectedClass}}">
```

Comment:
Using `<li>` leaves a dot next to the card, which is ugly. Better to use a `div` and use css grid to arrange cards in a list-like structure on screen